<launch>
    <node name="position_sensor" pkg="ssf_updates" type="position_sensor" clear_params="true" output="screen">
	    <remap from="position_sensor/hl_state_input" to="/mav1/fcu/ekf_state_out" />
	    <remap from="position_sensor/correction" to="/mav1/fcu/ekf_state_in" />

	    <remap from="position_sensor/position_measurement" to="/totalstation/position" />

	    <rosparam file="$(find ssf_updates)/position_sensor_fix.yaml"/>
    </node>
</launch>
